src/target/riscv/riscv-013.c

```diff
@@ -2918,6 +2918,10 @@ static int execute_fence(struct target *target)
 	if (dm013_select_target(target) != ERROR_OK)
```

Comment:
Could the function be renamed to `execute_autofence()` for improved clarity?

There is also function `maybe_execute_fence_i() ` which just calls `execute_autofence()`. Please, could you remove that function. Instead, call `execute_autofence()` directly.

Reply:
Thanks, addressed